app/views/smart_proxies/show.html.erb

```diff
@@ -38,6 +38,14 @@
             <%= @smart_proxy.url %>
           </div>
         </div>
+        <div class="row">
+          <div class="col-md-4">
+            <strong><%= _('UUID') %></strong>
+          </div>
+          <div class="col-md-8">
+            <%= @smart_proxy.uuid %>
+          </div>
+        </div>
```

Comment:
This should be probably handled like a password, e.g. not showing it on the page by default (screen recording etc). We do show provisioning UUID however on host detail page, that's also not good.

Reply:
By default or at all?

Reply:
I meant not showed like a bank PIN. Whatever.